sscanss/app/window/view.py

```diff
@@ -60,23 +61,32 @@ def __init__(self):
         self.readSettings()
         self.updateMenus()
 
+    def loadAppStyleSheet(self):
+        """loads the style sheet"""
```

Comment:
This does not load the theme. When I set the theme to dark then close and reopen the software the theme is back to light

Reply:
I updated the function so that the last theme selected is loaded on app start up.